duckduckgo/macos-browser
# Adds a file that's generated when the unit tests are run

Task/Issue URL: https://app.asana.com/0/0/1205311594925819/f

## Description:

Adds a file that's generated by our unit tests that's causing fastlane to fail.

## Steps to test this PR:

1. Just run the unit tests for our Developer ID and App Store tests.

Ensure no new file is created, and there are no modifications to existing files.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
